### PR TITLE
Refacto layouts

### LINKED ISSUE #296: Refacto Layouts
## Description

Replace `MainLayout` by specific `<q-layout>` for each page/route.
Update pages/components accordingly: For instance, place the `FileExplorer` in a `<q-drawer>`.
Rework on routes too.

Expect that ModelizerDrawView and others are renamed and use `q-page`.

### PR DESCRIPTION
**Pull request by k-y0u**
resolves #296

### CODE REVIEW

File: src/components/drawer/ModelizerTextLeftDrawer.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <default-drawer
+    :has-close-btn="false"
+    :bordered="false"
+    class="column no-wrap bg-grey-2"
+    data-cy="home-drawer"
```
Comment (justinehell) [inline src/components/drawer/ModelizerTextLeftDrawer.vue:6]: why not same name as component ?

**Review by justinehell — CHANGES_REQUESTED** [review]

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/drawer/ModelizerTextLeftDrawer.vue (hunk 1 above)
Comment (k-y0u) [reply src/components/drawer/ModelizerTextLeftDrawer.vue]: done

File: src/components/drawer/ModelizerTextLeftDrawer.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <default-drawer
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/components/drawer/ModelizerTextLeftDrawer.vue]: done

File: src/pages/ModelizerDrawPage.vue [hunk 3]
```diff
@@ -204,55 +138,17 @@ async function dropHandler(event) {
...
-  }
-  .modelizer-draw-view {
-    height: calc(100vh - 64px)
-  }
-</style>
-
-<style lang="scss">
-// Quasar sets overflow to 'hidden' on all svg.
-// In our case, it needs to be set to 'visible' to manage position with % in plugin models.
-  div#root svg {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/pages/ModelizerDrawPage.vue:236]: done

File: src/pages/ModelizerDrawPage.vue [hunk 4]
```diff
@@ -204,55 +138,17 @@ async function dropHandler(event) {
...
 });
 
 onUnmounted(() => {
-  pluginInitSubscription.unsubscribe();
   pluginDefaultSubscription.unsubscribe();
+  pluginInitSubscription.unsubscribe();
 });
 </script>
-
-<style scoped>
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/pages/ModelizerDrawPage.vue:223]: done

File: src/pages/ModelizerDrawPage.vue [hunk 5]
```diff
@@ -204,55 +138,17 @@ async function dropHandler(event) {
...
-  #app {
-    font-family: Avenir, Helvetica, Arial, sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    text-align: center;
-    color: #2c3e50;
-    margin-top: 60px;
-  }
-
-  #viewport {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/pages/ModelizerDrawPage.vue:252]: done

File: src/layouts/ModelizerDrawLayout.vue [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+/**
+ * Update plugin, draw components and update component templates array.
+ * @return {Promise<void>} Promise with nothing on success otherwise an error.
+ */
+async function initView() {
+  if (!query.value?.path) {
+    return;
+  }
+
+  const pluginName = query.value.path.split('/')[0];
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/layouts/ModelizerDrawLayout.vue]: done

**Review by justinehell — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**